ovs/action.go

```diff
@@ -385,6 +389,36 @@ type resubmitAction struct {
 	table int
 }
 
+// ResubmitPort resubmits a packet into the current table with its context modified
+// to look like it originated from the specified openflow port ID.
+func ResubmitPort(port int) Action {
+	return &resubmitPortAction{
+		port: port,
+	}
+}
+
+// A resubmitPortAction is an Action which is used by ConneectionTracking.
+type resubmitPortAction struct {
+	port int
+}
+
+// MarshalText implements Action.
+func (a *resubmitPortAction) MarshalText() ([]byte, error) {
+	// Largest valid port ID is 0xfeff per openflow spec.
+	if a.port < 0 || a.port > 0xfeff {
```

Comment:
That limitation is only for OpenFlow 1.0 right ? Afterward it is 0..0xfffffeff
If you want to only support OpenFlow 1.0 could that be added as a comment ?

Reply:
Ah yes, looks like I was.  Will need to update in a later PR